integ-tests/src/cucumberTest/java/cucumber/steps/helpers/InputCreator.java

```diff
@@ -20,95 +20,100 @@
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentRequest;
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentRequest;
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DescribeEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.ListClustersRequest;
 import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.UpdateEnvironmentRequest;
-
+import java.util.Collections;
+import java.util.StringJoiner;
 import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
 
 public class InputCreator {
-  private static final String ENVIRONMENT_NAME_PREFIX = "test";
-  private static final int ACCOUNT_ID_SIZE = 12;
-  private static final String ACCOUNT_ID = generateAccountId();
-  private static final String TASK_DEFINITION_ARN =
-      "arn:aws:ecs:us-east-1:" + ACCOUNT_ID + ":task-definition/sleep";
-  private static final String ROLE_ARN = "arn:aws:iam::" + ACCOUNT_ID + ":role/testRole";
-  private static final String CLUSTER_NAME_PREFIX = "cluster";
+  private static final String DEFAULT_ACCOUNT_ID = "123456789012";
+  private static final String DEFAULT_CLUSTER_NAME = "Cluster";
+  private static final String DEFAULT_ENVIRONMENT_NAME = "Environment";
+  private final String sharedPrefix = "blox-integ-tests";
```

Comment:
Should we make sharedPrefix static as well?

Reply:
Whoops, yes, will fix.